core/src/test/scala/org/apache/spark/deploy/history/ApplicationCacheSuite.scala

```diff
@@ -370,7 +370,9 @@ class ApplicationCacheSuite extends SparkFunSuite with Logging with MockitoSugar
 
     val request = mock[HttpServletRequest]
     when(request.getMethod()).thenReturn("GET")
-    when(request.getRequestURI()).thenReturn("http://localhost:18080/history/local-123/jobs/job/")
+    when(request.getRequestURI()).thenReturn("/history/local-123/jobs/job/")
```

Comment:
Ur, do we need this change? Usually, we try to avoid changing other test coverage.

Reply:
getRequestURI() was incorrectly mocked here. getRequestURI() does not return protocol or host name, only part of the path after and until the query parameters.

Reply:
Oh, was it? Thanks!